gr-filter/apps/gr_filter_design

```diff
@@ -20,7 +20,7 @@
 # Boston, MA 02110-1301, USA.
 #
 
-from gnuradio.filter import filter_design
```

Comment:
As far as I understand the referenced issue, the idea is to restructure the location of those files not the module structure.
Specifically, I suggest to keep gnuradio's module structure. Thus, `filter_design` should be located in `gnuradio.filter`. Otherwise functions that are specific to this tool spill into the gnuradio module. This would not be a good location for them.